src/conventional/amv_ssec_ascii2ioda.py

```diff
@@ -179,18 +186,25 @@ def read_file(file_name, data):
 
         for row in reader:
             try:
-                year = int(row['day'][0:4])
-                month = int(row['day'][5:6])
-                day = int(row['day'][7:8])
-                hour = int(row['hms'][0:1])
-                minute = int(row['hms'][2:3])
-                second = 0
-                dtg = datetime(year, month, day, hour, minute, second)
+                dtg = datetime.strptime(f"{row['day']} {row['hms']}", '%Y%m%d %H%M')
                 time_offset = np.int64(round((dtg - epoch).total_seconds()))
                 data['dateTime'] = np.append(data['dateTime'], time_offset)
```

Comment:
the merge conflict appears to be with the performance upgrade to this converter that was merged in (https://github.com/JCSDA-internal/ioda-converters/pull/934) I don't have permission to push any changes but lines 191-246 should be changed to:

 ```
                local_data['dateTime'] = np.append(local_data['dateTime'], time_offset)
                local_data['longitude'] = np.append(local_data['longitude'], float(row['lon']))
                local_data['latitude'] = np.append(local_data['latitude'], float(row['lat']))

                pres = float(row['pre'])*100.
                local_data['air_pressure'] = np.append(local_data['air_pressure'], pres)
                wdir = float(row['dir'])*1.0
                wspd = float(row['spd'])*1.0
                if (wdir >= 0 and wdir <= 360 and wspd >= 0 and wspd < 300):
                    uwnd, vwnd = meteo_utils.meteo_utils().dir_speed_2_uv(wdir, wspd)
                else:
                    uwnd = float_missing_value
                    vwnd = float_missing_value

                local_data['eastward_wind'] = np.append(local_data['eastward_wind'], uwnd)
                local_data['northward_wind'] = np.append(local_data['northward_wind'], vwnd)

                if row['type'] in known_freq.keys():
                    freq = known_freq[row['type']]
                else:
                    freq = float_missing_value
                    unk_freq.append(row['type'])
                local_data['sensorCentralFrequency'] = np.append(local_data['sensorCentralFrequency'], freq)

                if row['sat'] in known_sat.keys():
                    satid = known_sat[row['sat']]
                else:
                    satid = int_missing_value
                    unk_sat.append(row['sat'])

                local_data['satelliteIdentifier'] = np.append(local_data['satelliteIdentifier'], satid)
                local_data['sensorZenithAngle'] = np.append(local_data['sensorZenithAngle'], float(row['rff']))
                local_data['windTrackingCorrelation'] = np.append(local_data['windTrackingCorrelation'], float(row['qi']))
                local_data['windHeightAssignMethod'] = np.append(local_data['windHeightAssignMethod'], int(row['int']))

            except KeyError as e:
                if not keyerr:
                    logging.warning(file_name + ' is missing variable ' + e.args[0])
                keyerr = True

                if (e.args[0] == 'dir') or (e.args[0] == 'spd'):
                    local_data['eastward_wind'] = np.append(local_data['eastward_wind'], float_missing_value)
                    local_data['northward_wind'] = np.append(local_data['northward_wind'], float_missing_value)
                else:
                    local_data[known_var[e.args[0]][0]] = np.append(local_data[known_var[e.args[0]][0]], known_var[e.args[0]][1])

        for f in unk_freq:
            logging.warning(file_name + ' contains unknown frequency ' + f)

        for s in unk_sat:
            logging.warning(file_name + ' contains unknown satellite ID ' + s)

        if len(row.keys()) > len(known_var.keys()):
            for k in row.keys():
                if k not in known_var.keys():
                    logging.warning(file_name + ' contains unknown variable ' + k)

    for key in varDict.keys():
        data[key] = np.append(data[key], local_data[key])
    for key in meta_keys:
        data[key] = np.append(data[key], local_data[key])

```